sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -838,6 +845,20 @@ object SQLConf {
     .booleanConf
     .createWithDefault(false)
 
+  val ORC_META_CACHE_ENABLED = buildConf("spark.sql.orc.metadataCache.enabled")
+    .doc("To indicate if enable orc file meta cache, it is recommended to enabled " +
+      "this config in long-running process mode, such as Thrift Server, default is false")
+    .version("3.2.0")
+    .booleanConf
+    .createWithDefault(false)
+
+  val FILE_META_CACHE_TTL_SINCE_LAST_ACCESS =
+    buildConf("spark.sql.metadataCache.ttl.sinceLastAccess")
+      .version("3.2.0")
+      .doc("Time-to-live for file metadata cache entry after last access, the unit is seconds.")
+      .timeConf(TimeUnit.SECONDS)
+      .createWithDefault(1000L)
```

Comment:
This looks like a magic number. Why do we need to choose `1000` second as a default? Usually, we choose 1 hour (3600s), 1 day, or 1 weeks, don't we?

Reply:
change to use 1 hour (3600s)